front-end/map-search-test/Readme.md

```diff
@@ -5,25 +5,33 @@
 You've been given:
 
 1. `index.html`: The base HTML file with a full-page Google Map
-2. `main.css`: The base CSS file
-3. `main.js`: The base JS file
-4. `sample-data.js`: The sample data that should be searched. Each record has _at least_ an `id` and `name`.
+1. `main.css`: The base CSS file
```

Comment:
@mikefogg yes. It's a markdown feature. You don't have to number them; they get automatically incremented in translation.